lib/game_list_view/game_list_view.dart

```diff
@@ -53,26 +52,26 @@ class GameListView extends StatelessWidget {
     var controller = Provider.of<GameListController>(context, listen: false);
     return PopupMenuButton<String>(
       onSelected: (String value) {
-        if (value == imprintKey) {
-          showMarkdownPopup(context, imprintTitle, controller.imprint);
-        } else if (value == privacyKey) {
-          showMarkdownPopup(context, privacyTitle, controller.privacy);
-        } else if (value == creditsKey) {
+        if (value == GameListView.imprintKey) {
+          showMarkdownPopup(context, GameListView.imprintTitle, controller.imprint);
+        } else if (value == GameListView.privacyKey) {
+          showMarkdownPopup(context, GameListView.privacyTitle, controller.privacy);
+        } else if (value == GameListView.creditsKey) {

```

Comment:
Nachdem das jetzt wieder in derselben Kalsse ist, sind die `GameListView.` qualifier die überall hinzugefügt wurden nicht mehr nötig, und sollten wieder entfernt werden.